src/test/java/com/icthh/xm/ms/entity/service/MailServiceUnitTest.java

```diff
@@ -357,4 +362,41 @@ private void assertMultipart(List<Multipart> multiparts) throws Exception {
         byte[] arrayFromInputStream = IOUtils.toByteArray(dataSource.getInputStream());
         assertArrayEquals(FILE_BYTE_ARRAY, arrayFromInputStream);
     }
+
+    @Test
+    @SneakyThrows
+    public void testSendEmailWithContentAndAttachment()  {
+        prepareConfig(new HashMap<>() {{
+            put(MAIL_SETTINGS, singletonList(new HashMap<>()));
+        }});
+
+        MimeMessage mock = sendEmailWithContentAndAttachment();
+
+        verify(mock).setRecipient(eq(Message.RecipientType.TO),  eq(InternetAddress.parse(TO)[0]));
+        verify(mock).setFrom(eq(InternetAddress.parse(MOCK_FROM)[0]));
+        verify(mock).setSubject(eq(MOCK_SUBJECT), eq("UTF-8"));
```

Comment:
suggest verifying that text is equal to content provided as an input and differs from default template content

Reply:
Current realization doesn't allow us to do such test, due reason that in MailService we are using MimeMessageHelper which set content as mimepart.